src/openms_gui/source/VISUAL/SpectraViewWidget.cpp

```diff
@@ -444,87 +479,75 @@ namespace OpenMS
       std::map<Precursor, std::vector<Size>, Precursor::MZLess>& map_precursor_to_chrom_idx = map_precursor_to_chrom_idx_cache_[(size_t)(exp.get())];
       if (!was_cached)
       { // create cache: collect all precursor that fall into the mz rt window
-        for (std::vector<MSChromatogram >::const_iterator iter = exp->getChromatograms().begin(); iter != exp->getChromatograms().end(); ++iter)
+        for (auto it = exp->getChromatograms().begin(); it != exp->getChromatograms().end(); ++it)
```

Comment:
cbegin and cend? or did you change access patterns?

Reply:
good catch